get_github_coderabbit_review/coderabbit_fetcher/cli/main.py

```
@@ -1,121 +1,596 @@
+            self.resolution_request_manager = ResolutionRequestManager(
+                self.github_client, request_config
+            )
+
+    def validate_pr_url(self, pr_url: str) -> None:
+        """Validate PR URL format."""
+        try:
+            self.github_client.parse_pr_url(pr_url)
+        except InvalidPRUrlError as e:
+            print(f"❌ Invalid PR URL: {e}", file=sys.stderr)
+            print("\n💡 Expected format: https://github.com/owner/repo/pull/123", file=sys.stderr)
+            raise CLIError("Invalid pull request URL")
```

Ruff (0.13.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)